src/Initializer/Parameters/OutputParameters.cpp

```diff
@@ -41,7 +51,7 @@ CheckpointParameters readCheckpointParameters(ParameterReader* baseReader) {
          {"posix", CheckpointingBackend::POSIX},
          {"hdf5", CheckpointingBackend::HDF5},
          {"mpio", CheckpointingBackend::MPIO},
-         {"mpio_async", CheckpointingBackend::MPIO_ASYNC},
+         {"mpio_async", CheckpointingBackend::MpioAsync},
```

Comment:
I fully agree—however the code is obsolete right now anyways and will be removed by #1103 . Therefore I didn't really look further into it (otherwise, the whole enum would have been adjusted, probably).